src/Sepes.Infrastructure/Service/DataModelService/SandboxDatasetModelService.cs

```diff
@@ -68,9 +51,16 @@ AvailableDatasets MapToAvailable(Sandbox sandbox)
             return result;
         }
 
+        public async Task<List<SandboxDataset>> GetSandboxDatasetsForPhaseShiftAsync(int sandboxId)
+        {
+            var queryable = SandboxBaseQueries.SandboxDatasetForPhaseShift(_db);
+            queryable = queryable.Where(sds => sds.SandboxId == sandboxId);
+            return await queryable.ToListAsync();
```

Comment:
Is queryable a list of queries here? Since you have to put a where. Can you send sandboxId in on line 56? I might be out of the loop, but I though Id ask

Reply:
Might be the naming that is misleading, it's actually just a single queryable on the SandboxDataset table, in theory it could, but some might wish to use it in another way, without that sandbox filter